docs/AppService/GetAppServiceData/assets/GetAppServiceData_HMI_ASC.puml

```diff
@@ -0,0 +1,13 @@
+@startuml GetAppServiceData_HMI_ASC
+participant HMIConsumer
+participant Core
+participant MobileProvider
+
+MobileProvider -> Core: PublishAppService(MEDIA)
+Core --> MobileProvider: PublishAppService Response
```

Comment:
I don't know whether the PublishAppService is necessary here, it's really a precondition